pkg/ibmvpc/vpc.go

```diff
@@ -442,35 +509,45 @@ func (sgl *SecurityGroupLayer) RulesInConnectivity(src, dst vpcmodel.Node,
 		return nil, nil, fmt.Errorf("explainability for IKS node not supported yet")
 	}
 	for index, sg := range sgl.sgList {
-		sgRules, err1 := sg.RulesInConnectivity(src, dst, conn, isIngress)
+		tableRelevant, sgRules, err1 := sg.rulesInConnectivity(src, dst, conn, isIngress)
 		if err1 != nil {
 			return nil, nil, err1
 		}
-		if len(sgRules) > 0 {
+		if tableRelevant {
+			var rType vpcmodel.RulesType = vpcmodel.OnlyAllow
```

Comment:
```suggestion
			rType := vpcmodel.OnlyAllow
```

Reply:
no, I have to define the variable explicitly so that it will be of type `vpcmodel.RulesType` and not just `int`